src/Symfony/Bridge/Doctrine/Form/DoctrineOrmTypeGuesser.php

```diff
@@ -56,31 +57,32 @@ public function guessType($class, $property)
             return new TypeGuess('Symfony\Bridge\Doctrine\Form\Type\EntityType', ['em' => $name, 'class' => $mapping['targetEntity'], 'multiple' => $multiple], Guess::HIGH_CONFIDENCE);
         }
 
+        $useDeprecatedConstants = !class_exists(Types::class);
```

Comment:
@nicolas-grekas Is this variable useless?

Reply:
Maybe use a private property also?